internal/commands/user/list/flags.go

```diff
@@ -0,0 +1,65 @@
+package list
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
+	flagState      = "state"
+	flagStateShort = "s"
+	flagStateUsage = `select the state of users to list, available options: ["enabled", "disabled"]`
+
+	flagStatus      = "status-pending"
```

Comment:
still need to rename this flag value to `pending`, and can we also update the `var` names to be `flagPending...`